contacts_theme.theme

```diff
@@ -70,8 +70,53 @@ function contacts_theme_preprocess_block(&$variables) {
     ];
     foreach ($variables['content'] as $key => &$content) {
       if (in_array($key, $keys)) {
-        $content['button']['#value'] = '';
-        $content['button']['#attributes']['class'][] = 'button-action-icon';
+        $content['#attributes']['class'] = array('dropdown');
+        $content['#prefix'] = '<div class="action-group-wrapper dropdown">';
+        $content['#suffix'] = '</div>';
+        $drop_id = \Drupal\Component\Utility\Html::getUniqueId('dropdown-menu-button');
+        $content['button']['#attributes']['id'][] = $drop_id;
+        $content['button']['#attributes']['data-toggle'][] = 'dropdown';
+        $content['button']['#attributes']['aria-haspopup'][] = 'true';
+        $content['button']['#attributes']['aria-expanded'][] = 'false';
+        $content['button']['#attributes']['type'][] = 'button';
+        $content['button']['#attributes']['class'][] = 'btn';
+        $content['button']['#attributes']['class'][] = 'btn-lg';
+        $content['button']['#attributes']['class'][] = 'btn-primary';
+
+        // Replace text with icon.
+        if ($key == 'contacts.contact_create_group') {
```

Comment:
Pull this out of the `foreach` and do `$variables['content']['contacts.contact_create_group']`